src/test/java/ebl/config/TestConfig.java

```diff
@@ -0,0 +1,23 @@
+package ebl.config;
+
+import io.restassured.RestAssured;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class TestConfig {
+
+  private TestConfig() {}
+
+  public static void init() throws IOException {
+    File file = new File("src/test/resources/test.properties");
```

Comment:
Better use @ConfigurationProperties or @Value to retrieve properties

Reply:
I would but we do not use @SpringBootTest, so there is no spring context for it to be able to use @ConfigurationProperties or @Value. 
Ideally, we would use @SpringBootTest to run Integrations tests that would start a spring context and test the logic but since we build the containers here it seems redundant to have it start up a context (note that this context would look for properties related to the dependencies present)

Reply:
fair enough